tests/common/utilities.py

```diff
@@ -1492,3 +1492,13 @@ def cleanup_prev_images(duthost):
     current_os_version = duthost.shell('sonic_installer list | grep Current | cut -f2 -d " "')['stdout']
     duthost.shell("sonic_installer set-next-boot {}".format(current_os_version), module_ignore_errors=True)
     duthost.shell("sonic_installer cleanup -y", module_ignore_errors=True)
+
+
+def event_publish_tool(duthost, json_file='', count=1):
+    cmd = "docker exec eventd python /usr/bin/events_publish_tool.py"
+    if json_file == '':
+        cmd += " -c {}".format(count)
+    else:
+        cmd += " -f /{}".format(json_file)
```

Comment:
>/ [](http://example.com/codeflow?start=20&length=1)

Add '/' before caller provided path is weird. caller may provide a  relative path.

Reply:
Removed json file option